README.md

```diff
@@ -413,6 +413,30 @@ This would generate the following HTML:
 </div>
 ```
 
+### Container attributes
+
+Attributes of the outer control container can be changed via the `container` option, cutting the need to use custom
+templates for simple changes. The `class` attribute is a special case, its value will be prepended to the existing
+list of classes instead of replacing it.
+
+```php
+echo $this->Form->control('title', [
+    'container' => [
```

Comment:
Core provides [templateVars] options for custom template vars, so I would prefer we don't add more custom top level options https://book.cakephp.org/4/en/views/helpers/form.html#adding-additional-template-variables-to-templates

Reply:
I was kinda expecting that, so let me try to further make the case for this feature.

The point is being able to avoid having to use custom templates for as long as possible. Given how complex the templates already are, it would be nice if users could avoid them until they really need to make larger changes. I've caught myself once or twice already, deliberately avoiding customizing the templates, and instead resorting to wrapping the control in additional plain HTML, and while it's usually easy enough, avoiding that too would be nice.

Furthermore, dynamically adding attributes doesn't work with template variables alone, there would need to be additional logic that transforms the individual variables to a well formed string of HTML attributes.

If those arguments can't warm your heart, then I guess it just shouldn't be, and we'll have to continue customizing templates and extending helpers like the wild caveman that we are :neckbeard:

Reply:
I have a raging fire in my heart for bootstrap which wants to incinerate it, but I reluctantly continue using it as free backend themes for other libs are not easily available.

As long as I am not the one maintaining this options / templates soup, I'll just continue bitching about things from time to time and let you hold the torch :slightly_smiling_face:.

Reply:
This has been the worst trade deal in the history of trade deals, maybe ever!